isis/src/base/apps/pds2isis/pds2isis.xml

```diff
@@ -8,54 +8,69 @@ xsi:noNamespaceSchemaLocation=
+      does not need to be defined, unless the program fails to find the image file
       specifed in the label file upon the program's execution.
-      
-      The default settings normally work for most PDS products, but occasionally 
-      the user may need to modify the default values in order to obtain an image 
+
+      The default settings normally work for most PDS products, but occasionally
+      the user may need to modify the default values in order to obtain an image
       that meets their specific requirements.
      </p>
      <p>
-      The incoming <def link="Special Pixels">special pixel</def> values in the 
-      PDS file are assigned equivalent values for special pixels in ISIS3.  The 
-      order of precedence for special pixels in ISIS from highest to lowest 
-      priority is the following:  
+      The incoming <def link="Special Pixels">special pixel</def> values in the
+      PDS file are assigned equivalent values for special pixels in ISIS3.  The
+      order of precedence for special pixels in ISIS from highest to lowest
+      priority is the following:
       <blockquote>
       <ol>
-        <li><def>NULL</def></li> 
-	<li><def>HRS</def></li> 
-	<li><def>LRS</def></li> 
+        <li><def>NULL</def></li>
+	<li><def>HRS</def></li>
+	<li><def>LRS</def></li>
 	<li><def>HIS</def></li>
 	<li><def>LIS</def></li>
       </ol>
-      </blockquote> 
-      If any incoming <def link="Digital Number">pixel value</def> falls within 
-      two different special pixel types, the <def link="Special Pixel">special 
-      pixel</def> with the higher priority is assigned.  The user may also 
+      </blockquote>
+      If any incoming <def link="Digital Number">pixel value</def> falls within
+      two different special pixel types, the <def link="Special Pixel">special
+      pixel</def> with the higher priority is assigned.  The user may also
       specify a range of pixel values to be assigned to the different special pixels.
-      If the range of pixel values defined by the user for the different special 
-      pixels overlap, the special pixel with the highest priority is assigned.  For 
-      example, if NULLMIN=0.0, NULLMAX=3.0, LISMIN=3.0, and LISMAX=5.0, the actual 
-      raw value 3.0 can be assigned as a NULL or LIS, but is translated to NULL because 
-      NULL has a higher priority than LIS.  The conversion of images from ISIS2 to 
-      ISIS3 follows the same rules as PDS images, except that the label information 
+      If the range of pixel values defined by the user for the different special
+      pixels overlap, the special pixel with the highest priority is assigned.  For
+      example, if NULLMIN=0.0, NULLMAX=3.0, LISMIN=3.0, and LISMAX=5.0, the actual
+      raw value 3.0 can be assigned as a NULL or LIS, but is translated to NULL because
+      NULL has a higher priority than LIS.
+    <p>
+      It should be noted for 8-bit images
+      that ISIS uses zero as it's default NULL value so any
```

Comment:
You should also mention that 255 is also reserved for High Instrument Saturation.

Reply:
Something like: "It should be noted if any incoming DN values are equal to one of the ISIS special pixel (LINK) values for the corresponding bit type, those pixels will be special in the output cube."

Reply:
It would be nice to tell users exactly what those DNs are for each bit type. Do we have a link in our glossary that has the special pixel values in it?

Reply:
We would need to update the glossary entry for special pixels. Someone would also need to actually find what those values are. I looked around for a bit (no pun intended) but couldn't find anything concrete.

Reply:
Special Pixel values are defined here:

https://github.com/USGS-Astrogeology/ISIS3/blob/dev/isis/src/base/objs/SpecialPixel/SpecialPixel.h

Reply:
There is also a wiki page defining special pixel values, but the link is broken for some reason: https://usgs-astrogeology.github.io/ISIS3/gh-pages/Special_Pixels.html. I'll see if I can dig up the original on redmine.

Reply:
I migrated the original to https://github.com/USGS-Astrogeology/ISIS3/wiki/SpecialPixels. I think it would be useful to link to this.

Reply:
I've linked the wiki page in the docs. There was some discussion on gitter about this not being a great way to go about things but should work for now.